nemo/collections/tts/models/audio_codec.py

```diff
@@ -684,4 +684,11 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
         )
         models.append(model)
 
+        model = PretrainedModelInfo(
+            pretrained_model_name="audio_codec_low_frame_rate_22khz",
+            location="https://huggingface.co/nvidia/Low-Frame-rate-Speech-Codec/resolve/main/Low_Frame-rate_Speech_Codec_21Hz.nemo",
+            description="For details about this model please refer to the model card: https://huggingface.co/nvidia/Low-Frame-rate-Speech-Codec",
```

Comment:
I think our convention would be to name the .nemo file `audio_codec_low_frame_rate_22khz.nemo`

Reply:
Unfortunately, We can use this convention for hugging face models because currently, Nemo tries to load the checkpoint with the name Low-Frame-rate-Speech-Codec.nemo (name of the hugging face model card + ".nemo").

Reply:
I think to keep the Hugging face model conventions we should keep the model name as the name of the model card (" Low Frame-rate Speech Codec"). Because it would make it easier for the users to find the model. What do you think guys @anteju @rlangman ?

Reply:
In the definition of `from_pretrained()` https://github.com/NVIDIA/NeMo/blob/main/nemo/core/classes/common.py#L710 the logic is:

1. If there is a "/" in the name like f"{repo_name}/{model_name}", then attempt to download from HF. Otherwise, attempt to download from NGC.
2. If downloading from NGC, download using the information in the `list_available_models()`
3. If downloading from HF, then download file at hardcoded path f"{repo_name}/{model_name}/{model_name}.nemo".

The only models I saw on HF using this logic are Parakeet and Canary. I feel like the downloading logic should be more configurable.

But if we don't want to modify the logic ourselves, I would suggest following their convention of naming the model card all lowercase like "low-frame-rate-speech-codec-22khz". Though I would be alright with uppercase.

Reply:
Ok done, we opted to rename the model card to "low-frame-rate-speech-codec-22khz" to make it consistent with all codecs released and that we will release in the future.